src/components/templates/PartyMap/components/RoomModal/RoomModal.tsx

```diff
@@ -20,108 +20,121 @@ import { RoomModalOngoingEvent, ScheduleItem } from "..";
 
 import "./RoomModal.scss";
 
-interface RoomModalProps {
+export interface RoomModalProps {
   onHide: () => void;
+  show: boolean;
+  venue?: AnyVenue;
+  room?: Room;
+}
+
+export interface RoomModalContentProps {
   room: Room;
+  venueName: string;
 }
 
-export const RoomModal: React.FC<RoomModalProps> = ({ onHide, room }) => {
+export const RoomModal: React.FC<RoomModalProps> = ({
+  onHide,
+  room,
+  show,
+  venue,
+}) => (
+  <Modal show={show} onHide={onHide}>
+    <div className="container room-modal-container">
+      {room && venue && <RoomModalContent room={room} venueName={venue.name} />}
+    </div>
+  </Modal>
```

Comment:
It looks like we sort of keep the transition animation (so it seems it's applied to the container/similar), but because we're un-rendering the content within the modal, it looks like it quickly loses its height as part of the 'close modal' transition. We may want to give it a min-height, or do something else to 'solve this' in the future, but it doesn't seem like a big enough issue to block/delay merging this PR over. Would be good to backlog this for future fixing/similar.

Reply:
https://github.com/sparkletown/internal-sparkle-issues/issues/306